.env.production

```diff
@@ -2,6 +2,8 @@
 REACT_APP_SERVER_BASE_URL=https://www.anyway.co.il/
 # alternative url:
 # REACT_APP_SERVER_BASE_URL=https://anyway-mock-server.herokuapp.com
+REACT_APP_AUTH_SERVER_BASE_URL=https://anyway-mock-server.herokuapp.com
+
 
 REACT_APP_GOOGLE_LOGIN_LINK = https://anyway-mock-server.herokuapp.com/auth/google-login
```

Comment:
Please provide a standard naming (all environment files should have same variables, event if not always used)
Can be a separate issue